src/pages/Landing/index.tsx

```diff
@@ -402,8 +479,8 @@ export default function Landing() {
               />
             ))}
           </CardGrid>
-          <CardGrid cols={3}>
-            {MORE_CARDS.map(({ darkIcon, lightIcon, ...card }) => (
+          <CardGrid cols={renderUkSpecificText ? 2 : 3}>
```

Comment:
can this be less specific? like can we use `extraCards.length` to determine this value? 

it's fine with one parameter but i'm just imagining the cards logic getting more complex

Reply:
based